### DISCUSSION TITLE
Can I use MCU to connect to an open thread border router through UART?

### DISCUSSION THREAD

**Discussion by zyra102388** [openthread/openthread#10009] [Q&A] [answered]
I want to design a gateway with border router. But I don't want to use Raspberry Pi. I want to use an MCU (e.g. STM32F4) instead and communicate with the border router through UART. Is this possible? Does Open Thread have support for this?

**Comment by jwhui** [comment, marked as answer]
Yes, it is possible to have an MCU host implement OpenThread Border Router.

One example: https://openthread.io/codelabs/esp-openthread-hardware
